test/object.civet

```diff
@@ -774,6 +774,14 @@ describe "object", ->
       ({[y]: x[y]})
     """
 
+    testCase """
+      member access with expression
+      ---
+      {x[f(y)]}
+      ---
+      let ref;({[ref = f(y),ref]: x[ref]})
```

Comment:
I just tested this on Chrome and it seems that JavaScript doesn't allow the comma operator inside brackets like this?? It looks like we need to wrap in parentheses like so:

```js
let ref;({[(ref = f(y),ref)]: x[ref]})
```

Reply:
Thanks, great catch!